coreneuron/permute/node_permute.cpp

```diff
@@ -184,16 +263,45 @@ static void update_pdata_values(Memb_list* ml, int type, NrnThread& nt) {
                 int ixnew = p_target[ix];
                 *pd = ixnew + diam0;
             }
-        } else if (s == -5) {  // assume pointer to membrane voltage
+        } else if (s == -5) {  // POINTER
+            // assume pointer into nt._data. Most likely voltage.
+            // If not voltage, most likely same mechanism for all indices.
             int v0 = nt._actual_v - nt._data;
```

Comment:
I think that this should be moved inside the 
```
if (etype == voltage) {
...
}
```
which is at like 273 to avoid any confusion

Reply:
Moved the statement but not the comment. The latter seems relevant to the entire block